publicapi/device.md

```diff
@@ -236,8 +236,6 @@ The API exposes two methods for dealing with subnet routes:
 - Get routes: [`GET /api/v2/device/{deviceID}/routes`](#get-device-routes) to fetch lists of advertised and enabled routes for a device
 - Set routes: [`POST /api/v2/device/{deviceID}/routes`](#set-device-routes) to set enabled routes for a device
 
-<a name="device-get"></a>
```

Comment:
Ooh I see what you mean by remove them from the files. I think these are actually okay to keep, but no strong opinion either way.